### PR TITLE
Replace hardcoded value 20 with ray._RAY_ID_SIZE.

### PR DESCRIPTION
**Pull request by robertnishihara**
Following up on some cleanups discussed with @richardliaw.

### CODE REVIEW

**Comment by robertnishihara** [comment]
jenkins retest this please

File: python/ray/__init__.py [hunk 1]
```diff
@@ -63,6 +63,8 @@
 # Fix this.
 __version__ = "0.4.0"
 
+_RAY_ID_SIZE = 20
```
Comment (richardliaw) [inline python/ray/__init__.py]: should this go in `ray.constants`?

**Review by richardliaw — COMMENTED** [review]
sorry for the late review - also conflict in tests

**Comment by richardliaw** [comment]
@robertnishihara are you still working on this?

**Comment by robertnishihara** [comment]
Thanks @richardliaw, I'll do it as a separate PR, since that will be easier than rebasing.

**Pull request closed without merging**